qtodotxt/ui/controllers/tasks_list_controller.py

```diff
@@ -201,7 +201,8 @@ def _recurTask(self, task):
         # Set new due date in old task text
         rec_text = task.updateDateInTask(task.text, next_due_date)
         # create a new task duplicate
-        return self.createTask(rec_text)
```

Comment:
Using this function (i found out there are two different createtask functions - one interesting the tasklib and one in the taskListController) the createTask function of the task list controller is used. The createTask function in the tasklib uses the popup editor.
I just saw that the createTask implementation in the taskListController is just what i need and uses obviously (it works in master) the new inline editing.

That also means the createTask function in the taskLib might be redundant. But i have to double-check that.